Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM python:3.7-slim-stretch
```

Comment:
not sure what this file is doing in this PR, but happy to explain how we use Docker if you'd like to build for the files which are committed.